spec/requests/get_involved_spec.rb

```diff
@@ -0,0 +1,15 @@
+RSpec.describe "Get Involved" do
+  before do
+    stub_content_store_has_item("/government/get-involved", GovukSchemas::Example.find("get_involved", example_name: "get_involved"))
+    stub_request(:get, /\A#{Plek.new.find('search-api')}\/search.json/).to_return(body: { "results" => [], "total" => 83 }.to_json)
```

Comment:
This is much better! Thanks for getting rid of the extraneous stubbing in this test 🎉